scripts/wheels/linux/configure.sh

```diff
@@ -72,4 +72,5 @@ ${CMAKE} -H"${KRATOS_SOURCE}" -B"${KRATOS_BUILD}/${KRATOS_BUILD_TYPE}" \
 -DBLAS_LIBRARIES="/usr/lib64/libblas.so.3"                             \
 -DINCLUDE_MMG=ON                                                       \
 -DMMG_ROOT="/workspace/external_libraries/mmg/mmg_5_4_1"               \
+-DKRATOS_BUILD_TESTING=OFF                                             \
```

Comment:
this should probably be the default for release builds

Reply:
yep. it is now :)